### ISSUE TITLE
Google Calendar makes `.ics`-imported events public

### ISSUE THREAD

**Issue by calejvaldez** [icssc/AntAlmanac#963]
When I add the events from AntAlmanac to my Google Calendar, it automatically makes the events public. I don't know if this is an AA issue or a GC issue, but I think it raises serious privacy concerns.

**Comment by MinhxNguyen7** [comment]
@calejvaldez Thanks for the notice. We never thought about this feature, and we'll look into it.

**Comment by calejvaldez** [comment]
I ended up figuring this out myself! So, turns out, it was both an AntAlmanac and Google Calendar issue.

AntAlmanac uses the [ics package](https://github.com/adamgibbons/ics) to generate its `.ics` files. However, when creating the event attributes, `classification` wasn't specified, which [specifies access to the event](https://icalendar.org/iCalendar-RFC-5545/3-8-1-3-classification.html).

According to the iCalendar specs, the default value is `PUBLIC`. However, [according to Google's Calendar API](https://developers.google.com/calendar/api/v3/reference/events/import), the default event visibility is the calendar's default visibility. (Though I'm sure Google doesn't internally use the public Calendar API, it's an indicator as to how they implement a missing `classification`.) I guess would be fine if a user specifies the default calendar visibility since it indicates a preference, but **UCI sets the default calendar's visibility as public**. I didn't know this until a few years ago when I was tinkering with my Calendar settings. This made me think, how many people are adding events to their UCI calendar without knowing they're fully public?

I made a fix in #1003, which sets the default `classification` to `PRIVATE`. I think that a user's events should be private unless they specify otherwise, but because UCI makes our events public without us knowing, then enforcing `PRIVATE` is probably a good idea.
